### PR TITLE
Deprecate merge_schedule

### PR DESCRIPTION
**Pull request by berland**
It  is a too thin wrapper around sunsch to deserve its own endpoint.

### CODE REVIEW

**Comment by berland** [comment]
The user are now faced with this when she runs the script:14:49 $ merge_schedule -v merge2.sch schedule.sch  foo.sch
```
......merge_schedule/merge_schedule.py:54: FutureWarning: merge_schedule is deprecated, use sunsch
  warnings.warn("merge_schedule is deprecated, use sunsch", FutureWarning)
Add the following lines to a yaml file and use it as input for sunsch:
----
files:
- merge2.sch
- schedule.sch
output: foo.sch
----
```

File: src/subscript/merge_schedule/merge_schedule.py [hunk 1]
```diff
@@ -50,14 +50,20 @@ def main():
...
+
+    sunsch_config = {"files": args.inputfiles}
 
     if args.verbose:
         # Set the root logger to INFO, will be inherited by sunsch
         logger.setLevel(logging.INFO)
 
-    logger.info("# Sending the following YAML configuration to sunsch:")
-    logger.info(yaml.dump(sunsch_config))
+    warnings.warn
```
Comment (anders-kiaer) [inline src/subscript/merge_schedule/merge_schedule.py:61]: What does this line do?

**Review by anders-kiaer — APPROVED** [review]

File: src/subscript/merge_schedule/merge_schedule.py (hunk 1 above)
Comment (berland) [reply src/subscript/merge_schedule/merge_schedule.py:61]: That should not be there!

Comment (berland) [reply src/subscript/merge_schedule/merge_schedule.py:61]: lgtm.com, where were you..? ;)

**Pull request merged**